packages/bun-types/package.json

```diff
@@ -16,7 +15,7 @@
   ],
   "homepage": "https://bun.sh",
   "dependencies": {
-    "@types/node": "*",
+    "@types/node": "^22.13.5",
```

Comment:
Can we keep this to a `*`? What if they use 23?

Reply:
Mistake, updated